playbooks/roles/openconnect/templates/instructions-fr.md.j2

```diff
@@ -62,11 +62,11 @@ Les certificats clients sont un mécanisme par lequel les clients peuvent s'auth
    `brew install openconnect`
 
    * Si l'installation de Homebrew n'est pas une option, vous pouvez également télécharger et compiler le [code source OpenConnect](/mirror/index-fr.html#openconnect).
-1. Téléchargez le fichier [ca-cert.pem](/openconnect/ ca-cert.pem) et un [fichier de certificat client](#clientcerts) dans la liste ci-dessus.
+1. Téléchargez le fichier [ca.crt](/openconnect/ ca.crt) et un [fichier de certificat client](#clientcerts) dans la liste ci-dessus.
```

Comment:
I think a space slipped into this URL: `[ca.crt](/openconnect/ ca.crt)` vs `[ca.crt](/openconnect/ca.crt)`